docs/docs/virtual.md

```diff
@@ -40,13 +41,27 @@ oisst_files = sorted(['s3://'+f for f in oisst_files])
 #]
 ```
 
+These are netCDF4 files, which are really HDF5 files, so we need to user virtualizarr's `HDFParser`.
+
+We also need to give the parser a way to access our files. We do this by creating a `ObjectStoreRegistry` containing an obstore `S3Store` for that bucket.
+
 Now that we have the filenames of the data we need, we can create virtual datasets with `VirtualiZarr`. This may take a minute.
 
 ```python
 from virtualizarr import open_virtual_dataset
+from virtualizarr.parsers import HDFParser
+from virtualizarr.registry import ObjectStoreRegistry
+
+bucket = "noaa-cdr-sea-surface-temp-optimum-interpolation-pds/"
+store = S3Store(
+    bucket=bucket,
+    region="us-west-2",
```

Comment:
I'm not sure this is the right region - @mpiannucci presumably you know?

Reply:
I think it's actually `us-east-2` https://registry.opendata.aws/noaa-cdr-oceanic/